sbc_compassion/models/sponsorship_correspondence.py

```diff
@@ -259,6 +265,37 @@ def _set_partner_review(self):
             if letter.correspondant_id.mandatory_review:
                 letter.mandatory_review = True
 
+    @api.depends('page_ids')
+    def _compute_page(self):
+        self.original_text = self._get_original_text('original_text')
+        self.translated_text = self._get_original_text('translated_text')
+        self.nbr_pages = len(self.page_ids)
+
+    def _inverse_page(self):
```

Comment:
Either add `@api.one` decorator or add a loop `for correspondence in self` and compute the result for each letter (self can be a recordset of many letters)